### PR TITLE
replace latent heat

### PR DESCRIPTION
**Pull request by yairchn**
This PR replaces Latent heat only throughout the code

### CODE REVIEW

File: src/Surface.jl [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import Thermodynamics
```
Comment (charleskawczynski) [inline src/Surface.jl:1]: I think this, and the next line, are defined in `src/TurbulenceConvection.jl`, so this is probably not needed.

File: src/microphysics_functions.jl [hunk 2]
```diff
@@ -1,18 +1,21 @@
 
+import Thermodynamics
```
Comment (charleskawczynski) [inline src/microphysics_functions.jl:2]: This and next line are needed— we only need it once in `src/`

**Comment by yairchn** [comment]
@charleskawczynski  do you know what all tests are red here ? CI and all ?

**Review by trontrytel — APPROVED** [review]
The artifact plots look reasonable. The CI is broken, but it does not look to be related to this PR.

**Pull request closed without merging**